cmd/manager/main.go

```diff
@@ -84,7 +84,14 @@ func main() {
 	}()
 	tracer := tp.Tracer(svcName)
 
-	qemuCfg := qemu.Config{}
+	qemuCfgPtr, err := qemu.NewConfig()
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to create config: %v", err))
+		exitCode = 1
+		return
+	}
+	qemuCfg := *qemuCfgPtr
```

Comment:
no need for this line

Reply:
also a log here after config is set, something like manager started with TDX enabled etc